client/app/dbaas/logs/detail/home/capped/logs-home-capped.modal.controller.js

```diff
@@ -0,0 +1,33 @@
+class LogsHomeCappedModalCtrl {
+  constructor($scope, $state, $stateParams, ControllerHelper) {
+    this.$scope = $scope;
+    this.$state = $state;
+    this.serviceName = $stateParams.serviceName;
+    this.ControllerHelper = ControllerHelper;
+    this.openModal();
+  }
+
+  openModal() {
+    this.ControllerHelper.modal.showModal({
+      modalConfig: {
+        templateUrl: 'app/dbaas/logs/detail/home/capped/logs-home-capped.html',
+        controller: 'LogsHomeCappedCtrl',
+        controllerAs: 'ctrl',
+        backdrop: 'static',
+      },
+    })
+      .then(() => {
+        this.$scope.$parent.$parent.$parent.$$prevSibling.ctrl.runLoaders();
```

Comment:
@FredericEspiau Thanks to pointing out this issue. As @pdepaepe said he just took them as sample from other part from the existing codebase.

We all agree that we have to refactor these part.

In order to move forward I will merge this pull request since it's now in "Quality Check Ok".

Thanks @pdepaepe.